pandas/tests/plotting/test_boxplot_method.py

```diff
@@ -160,6 +160,13 @@ def test_boxplot_empty_column(self):
         df.loc[:, 0] = np.nan
         _check_plot_works(df.boxplot, return_type='axes')
 
+    @slow
+    def test_figsize(self):
+        df = DataFrame(np.random.rand(10, 5), columns=['A', 'B', 'C', 'D', 'E'])
```

Comment:
this line might be a bit longn too. `flake8 pandas/tests/plotting/test_boxplot_method.py` should tell you.